lib/src/user_avatar.dart

```diff
@@ -38,7 +38,9 @@ class UserAvatar extends StatelessWidget {
           decoration: BoxDecoration(
             color: StreamChatTheme.of(context).accentColor,
           ),
-          child: user.extraData?.containsKey('image') ?? false
+          child: (user.extraData?.containsKey('image') == true &&
```

Comment:
Minor nitpick but you could extract the boolean logic into a variable at the top of the build function then use that to keep the tree clean. 

```
final bool hasImage = user.extraData?.containsKey('image') == true &&
                   user.extraData['image'] != null &&
                   user.extraData['image'] != '';
```
then 
```
hasImage
        ? CachedNetworkImage(		              
                   imageUrl: user.extraData['image'],		         
                   errorWidget: (_, __, ___) {
```